_targets.R

```diff
@@ -6,11 +6,44 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+# define NWIS stations of interest and file out paths
+nwis_stn <- c("01427207", "01432160", "01436690", "01466500") # removing "01435000" because it returns an unexpected column name
+fileout_stn <- paste0("1_fetch/out/nwis_", nwis_stn, ".csv")
+nwis_target <- paste0("nwis_", nwis_stn)
+
 p1_targets_list <- list(
+  # download data
+  tar_target(
+    nwis_target[1],
+    download_nwis_data(site_no = nwis_stn[1], pathout = "1_fetch/out/"),
```

Comment:
This is a nice implementation of "dynamically " building the target names. Similar to other comment, this makes scaling a little easier but you'll cover this with the dynamic branching in targets-3 training. Here, it's probably more interpretable and easier for the reader to see the actual target names then having to construct them in their head. USGS's standard practice is to have the target name be the output object or file. So for your first site target you would have `nwis_01427207_csv` as the target name. This will also make it easier to connect what shows up as the target names in the targets build status messages to built targets.